src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,141 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+    private Node<T> last;
+    private int size;
+
     @Override
     public boolean add(T value) {
-        return false;
+        if (isEmpty()) {
+            Node<T> node = new Node<>(value, null, null);
+            first = node;
+            last = node;
```

Comment:
let's move `last = node;` after if-else construction

Reply:
But why? I can write like this first = last = node;

Reply:
Or do you mean initialize in the constructor last and first?

Reply:
I did, thanks.